docs/rootless.md

```diff
@@ -23,29 +23,40 @@ penguin:231072:65536
 
 ### Distribution-specific hint
 
+Using Ubuntu kernel is recommended.
+
 #### Debian (excluding Ubuntu)
 * `sudo sh -c "echo 1 > /proc/sys/kernel/unprivileged_userns_clone"` is required
+* [`sudo modprobe overlay permit_mounts_in_userns=1` is required to use `overlay2` storage driver (recommended)](https://salsa.debian.org/kernel-team/linux/blob/283390e7feb21b47779b48e0c8eb0cc409d2c815/debian/patches/debian/overlayfs-permit-mounts-in-userns.patch)
```

Comment:
Perhaps create a separate link, instead of the whole line 🤔

Something along the line of (not sure if my caption is correct);

```suggestion
* `sudo modprobe overlay permit_mounts_in_userns=1` is required to use `overlay2` storage driver (recommended), which was [added in kernel 5.2-rc7](https://salsa.debian.org/kernel-team/linux/blob/283390e7feb21b47779b48e0c8eb0cc409d2c815/debian/patches/debian/overlayfs-permit-mounts-in-userns.patch)
```

Reply:
It is Debian-specific patch and not merged into the upstream.